packages/core/addon/utils/classes/interval.ts

```diff
@@ -4,31 +4,42 @@
  */
 
 import { assert } from '@ember/debug';
-import { A } from '@ember/array';
-import moment from 'moment';
+import moment, { Moment } from 'moment';
 import Duration from './duration';
 import { isIsoDurationString } from './duration';
-import DateUtils from '../date';
 import DurationUtils from '../duration-utils';
+import { API_DATE_FORMAT_STRING, getIsoDateTimePeriod, Grain } from '../date';
 
 const CURRENT = 'current';
 const NEXT = 'next';
-const MACROS = A([CURRENT, NEXT]);
+const MACROS = <const>[CURRENT, NEXT];
+
+function isMacro(property: unknown): property is typeof MACROS[number] {
+  const allMacros: string[] = [...MACROS];
+  return typeof property === 'string' && allMacros.includes(property);
+}
+
+type Serialized<T extends Moment | string> = { start: T; end: T | undefined };
```

Comment:
Do we use this for anything other than `Serialized<Moment>`?

Reply:
Nope good catch